pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.4
+lockfileVersion: 5.3
```

Comment:
Not really related to the aim of this PR, but noticed this lock file seems to have some odd side effect here, with the downgrade of version, and with various `dev: false` additions down the line. 

Not 100% sure what happened, but maybe we can just omit this file from this PR? Since this PR doesn't add/remove any dependencies.

Reply:
I think @Nitwel's local `pnpm` is outdated, which would cause the lockfile to update like that 🤔